scripts/EN/Harry

```diff
@@ -2623,18 +2623,18 @@ TitleAndSubtitle(0x3c, 0x230); // 0x30e8: 00002062
 Area(0x3c); // 0x30f0: 00001026
 PlayMusic(0x66, 0x1); // 0x30f6: 0000206c
 LoadSecondEffect(0x1f, 0x1, 0x1, 0x1); // 0x30fe: 0000406e
-ShowText((1213)"(Effortlessly, the arrow flew, piercing the defenseless maw, and the lizard fell down dead.)"); // 0x310a: 0000100c
+ShowText((1213)"(Effortlessly, W2the arrow flew, W2piercing the defenseless maw, W2and the lizard fell down dead.)"); // 0x310a: 0000100c
 HandleInput(); // 0x3110: 00000010
-ShowText((1214)"All I can do is draw my opponent closer. Each bullet that hits its body only seems to increase its strength."); // 0x3114: 0000100c
+ShowText((1214)"All I can do is draw my opponent closer.¥nW2Each bullet that hits its body W1only seems to increase its strength."); // 0x3114: 0000100c
 HandleInput(); // 0x311a: 00000010
-ShowText((1215)"I am running. My entire body is soaked in sweat. The heat from the flames and the large amount of perspiration are working together to rob me of my emotional strength. And then my feet finally get tangled and I fall to the bone-dry floor with a thud."); // 0x311e: 0000100c
+ShowText((1215)"I am running.¥nW2My entire body is soaked in sweat.¥nW2The heat from the flames and the large amount of perspiration are working together to rob me of my emotional strength.¥nW2And then my feet finally get tangled W1and W1I fall to the bone-dry floor with a thud."); // 0x311e: 0000100c
 HandleInput(); // 0x3124: 00000010
 ShowText((9998)""); // 0x3128: 0000100c
 LoadSecondEffect(0x1f, 0x2, 0x1); // 0x312e: 0000306e
 WaitTime(0x96); // 0x3138: 0000101f
-ShowText((1216)"My supply of bullets is running out. This is my limit, I turn around. Its enormous mouth is there, measuring about 4 meters, and open in all directions. It was only for an instant, but I thought of that passage again."); // 0x313e: 0000100c
+ShowText((1216)"My supply of bullets is running out.¥nW2This is my limit, W1I turn around.¥nIts enormous mouth is there, measuring more than 10 feet, W1and open in all directions. ¥nW2It was only for an instant, W1but I thought of that passage again."); // 0x313e: 0000100c
```

Comment:
Changed "4 meters" to "more than 10 feet" for the Americans. Technically 4 meters is 13 feet, but since Harry is just guessing a little less precision makes sense. Plus Victor and I had this same thought!